tests/realtime/utils.js

```diff
@@ -206,32 +206,9 @@ module.exports = {
     return JSON.parse(JSON.stringify(standardBotData));
   },
 
-  forceDeleteAspectsAndSubjects(done) {
-    tu.db.Event.destroy({
-      where: {
-        createdAt: {
-          $lt: new Date(),
-          $gte: testStartTime,
-        },
-      },
-      force: true,
-    })
-    .then(() => tu.forceDelete(tu.db.Subject, testStartTime))
-    .then(() => tu.forceDelete(tu.db.Aspect, testStartTime))
-    .then(() => done())
-    .catch(done);
-  },
-
   forceDelete(done) {
-    tu.db.Event.destroy({
```

Comment:
Probably should keep deleting Event, but use the tu.forceDelete helper fn instead of calling destroy explicitly here -- I imagine Tausif or someone from IMC added it for a reason